presto-product-tests/README.md

```diff
@@ -237,17 +238,25 @@ setup outlined below:
     presto-product-tests/conf/docker/singlenode/compose.sh logs
     ```
     
-3. Add an IP-to-host mapping for the `hadoop-master` host in `/etc/hosts`.
+3. Add an IP-to-host mapping for the docker hosts in `/etc/hosts`.
 The format of `/etc/hosts` entries is `<ip> <host>`:
 
-    - On GNU/Linux add the following mapping: `<container ip> hadoop-master`.
-    The container IP can be obtained by running:
+    - On GNU/Linux run the following command add output of below command to `/etc/hosts`: 
 
         ```
-        docker inspect $(presto-product-tests/conf/docker/singlenode/compose.sh ps -q hadoop-master) | grep -i IPAddress
+        presto-product-tests/bin/hosts.sh singlenode
```

Comment:
This script requires yet another external dependency. Personally i would prefer to have here simple instructions, like 

```
add

localhost mysql postgres
<container ip> hadoop-master

to /etc/hosts
```

But script has its own benefits. So, i don't have strong opinion here.

Also i know that this is not a scope of this PR, but i think we should change the ports of `mysql` and `postgree` from defaults to some specific. Mysql and Postgree ofthen are already installed on the host machine.  Than we should change the `mysql` and `postgres` hosts as well. Those names looks too generic.